development/auto-changelog.sh

```diff
@@ -1,26 +1,63 @@
-#! /bin/bash
-# update tags
+#!/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

macOS doesn't have `/bin/env` but some Linux-es do? 🤷🏽‍♂️

Reply:
Oops, yes that was a mistake.